src/services/featurehelper.js

```diff
@@ -88,6 +89,13 @@ ngeo.FeatureHelper = function($injector, $filter) {
    */
   this.projection_;
 
+  /**
+   * Download service.
+   * @type {ngeo.Download}
+   * @private
+   */
+  this.download_ = $injector.get('ngeoDownload');
```

Comment:
There isn't a clear guideline in ngeo when to use `$injector` and when to use automatic injection.